contrib/kakoune.spec

```diff
@@ -83,6 +83,9 @@ popd
 
 
 %changelog
+* Thu Jan 16 2020 Jiri Knoceny <jkonecny@redhat.com> - 2020.08.04-1
```

Comment:
Fully aware of the irony of correcting the author's name but thought I'd ask anyway, is the last name a typo?

Reply:
:D :+1: Good one!

Reply:
Even the date was wrong. Should be fine now.